components/eam/src/control/wrap_nf.F90

```diff
@@ -266,16 +266,16 @@ subroutine wrap_inq_varid (nfid, varname, varid, abort)
    integer, intent(in):: nfid
    integer, intent(out):: varid
    character*(*), intent(in):: varname
-   logical, optional :: abort
+   logical, optional :: abrtf
```

Comment:
Renamed to avoid `use associated / must not be redeclared` errors
```
ftn-855 crayftn: ERROR WRAP_NF, File = E3SM/components/eam/src/control/wrap_nf.F90, Line = 12, Column = 8
  The compiler has detected errors in module "WRAP_NF".  No module information file will be created for this module.


ftn-923 crayftn: ERROR WRAP_INQ_VARID, File = E3SM/components/eam/src/control/wrap_nf.F90, Line = 256, Column = 53
  "ABORT" has been use associated, therefore it must not be redeclared as a dummy-argument.


ftn-922 crayftn: ERROR WRAP_INQ_VARID, File = E3SM/components/eam/src/control/wrap_nf.F90, Line = 269, Column = 25
  "ABORT" has been use associated, therefore it must not be redeclared with the LOGICAL attribute.


ftn-922 crayftn: ERROR WRAP_INQ_VARID, File = E3SM/components/eam/src/control/wrap_nf.F90, Line = 269, Column = 25
  "ABORT" has been use associated, therefore it must not be redeclared with the OPTIONAL attribute.

Cray Fortran : Version 10.0.3 (20200807200759_6f9d38efdd5407d5b9bb59bc9fd249c0519c1858)
Cray Fortran : Compile time:  0.0425 seconds
Cray Fortran : 921 source lines
Cray Fortran : 4 errors, 2 warnings, 0 other messages, 0 ansi
Cray Fortran : "explain ftn-message number" gives more information about each message.
```